DOLConfig/DOLConfig.csproj

```diff
@@ -67,8 +67,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System.Windows.Forms" />
-    <PackageReference Include="MySql.Data" Version="6.10.9" ExcludeAssets="Runtime" />
-    <PackageReference Include="System.Data.SQLite.Core" Version="1.0.113.6" ExcludeAssets="All" Condition=" '$(OS)' != 'Unix' " />
+    <PackageReference Include="MySql.Data" Version="8.0.25" ExcludeAssets="Runtime" />

```

Comment:
Glad to see they finally fixed the performance regression in https://bugs.mysql.com/bug.php?id=101714